res/css/style.css

```diff
@@ -220,45 +219,52 @@ body, #root, .profileViewer {
   color: transparent;
 }
 
-.treeRowToggleButton {
-  background-image: url(./treetwisty.svg);
-}
-
 .focusCallstackButton {
-  background-image: url(./circlearrow.svg);
+  background-image: url(../img/svg/circlearrow.svg);
   margin-left: 5px;
   visibility: hidden;
 }
 
-.treeRowToggleButton:active:hover,
 .focusCallstackButton:active:hover {
   background-position: -16px 0;
 }
 
-.treeRowToggleButton.collapsed {
-  background-position: 0 -16px;
+.treeViewBody:focus .treeViewBodyInner .treeViewRow.selected > .focusCallstackButton {
+  background-position: -32px 0;
 }
 
-.treeRowToggleButton.collapsed:active:hover {
-  background-position: -16px -16px;
+.treeViewBody:focus .treeViewBodyInner .treeViewRow.selected > .focusCallstackButton:active:hover {
+  background-position: -48px 0;
 }
 
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .treeRowToggleButton,
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .focusCallstackButton {
-  background-position: -32px 0;
+
+.treeRowToggleButton {
+  height: 0;
+  width: 0;
+  color: #888;
+  border-left: 8px solid;
+  border-top: 5px solid transparent;
+  border-bottom: 5px solid transparent;
+  border-right: 3px solid transparent;
+  margin-left: 5px;
+  margin-right: 5px;
 }
 
-.treeViewBody:focus > * > * > * > .treeViewRow.selected > .treeRowToggleButton.collapsed {
-  background-position: -32px -16px;
+.treeRowToggleButton.expanded {
+  transform: rotate(90deg);
+  margin-top: 5px;
 }
```

Comment:
In the expanded state and at some zoom level (here at the default zoom level 0) we see a small line between the 2 parts of the twisty arrow:
![capture d ecran de 2018-05-04 15-06-43](https://user-images.githubusercontent.com/454175/39629566-efee7142-4fac-11e8-8289-9054703d862e.png)

I'm not sure what causes this and how this can be overcome...

Reply:
I can verify. This happens due to pixel calculation during zooming. So basically its painted not correctly as there is an additional pixel step that can not be interpreted correctly.

Not sure what would be the best way to fix it. @mstange @gregtatum any ideas?

Reply:
So I did some tries on my side, and I found that if `border-right`'s width == `border-left`'s width, we don't have the issue. So maybe we can set `border-right` at `8px` and use some clever negative margin to position it properly ?

I'll file a gecko bug.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1461680

Reply:
What about re-ordering the border arguments instead of doing a transform?

```
.treeRowToggleButton.expanded {
    border-top: 8px solid;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-bottom: 4px solid transparent;
}
.treeRowToggleButton {
    height: 0;
    width: 0;
    color: #888;
    border-left: 8px solid;
    border-top: 5px solid transparent;
    border-bottom: 5px solid transparent;
    border-right: 4px solid transparent;
    margin-left: 5px;
    margin-right: 5px;
    background-color: #888;
}
```

Reply:
Thanks @gregtatum 

@julienw Ready for (hopefully last) round of review :)

Reply:
Good suggestion ! Thanks Greg ! (I'll review tomorrow but I'm hopeful :) )